### PR TITLE
99 feature request display files in stash directory

### LINKED ISSUE #99: Feature Request: Display files in `stash` directory
Add a method `ls()` inside `show.py` that displays the files list inside the `stash` directory when executed.

**Example:**

```python

from package.show import display
display()
```
**Output**:
```
1. file1
2. file2
3. file3
4. file4
5. file5
6. file6
7. file7
8. file8
```

**Explanation**:
- As you can see the above code, it doesn't explicitly call the `ls()` function.
- The feature should execute `ls()` function when there isn't any arguments passed.

_In simple terms_, 
- Implement `ls()` method, that list files inside `stash` directory in python execution console.
- It should execute when `display()` is called without any arguments.

### PR DESCRIPTION
**Pull request by Prakhar29Sharma**
### Title of the Pull Request
- [x] Have you renamed the PR Title in a meaningful way?

### Related Issue
closes #99 

### Description
- Implemented ls() to display files as a numbered list inside the stash directory.
- Updated display() to call ls() when no snippet_name is passed.
- Ensured the output format matches the requirement of numbered file listing.
- Improved error handling with specific messages for FileNotFoundError, PermissionError, and unexpected exceptions.
- Checked read permissions for the stash directory before attempting to list files.
- Enhanced display() to handle errors while copying snippet files.
- Resolves issue #[issue number] with more robust error handling and user feedback.

### Type of change

What sort of changes have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist
- [x] My code follows the code style of this project.
- [x] I have followed the contribution guidelines
- [x] I have performed a self-review of my own code.
- [x] I have ensured my changes don't generate any new warnings or errors.
- [ ] I have updated the documentation (if necessary).
- [x] I have resolved all merge conflicts.

### CODE REVIEW

**Review by techy4shri — CHANGES_REQUESTED** [review]
Hello @Prakhar29Sharma
First and foremost, never edit files which are not part of your assigned issue. That is a violation ofcontribution guidelines in general and is considered point-farming.
As per your approach, you were to edit models.py file but here I can see numerous unnecessary file changes.
The snippets that you mentioned here, explain it.
The absence of models.py as mentioned in your first approach is extremely telling. I would urge you to check out the documentation of the project and opensource in general before making redundant pull requests.
Unnecessary changes and commits degrade the repository and will not be accepted. Keep this in mind while contributing to any repository in future.
@this-is-yaash you can close this PR as it lacks the solution and is simply additional of redundant code.

**Comment by Prakhar29Sharma** [comment]
> Hello @Prakhar29Sharma
> First and foremost, never edit files which are not part of your assigned issue. That is a violation ofcontribution guidelines in general and is considered point-farming.
> As per your approach, you were to edit models.py file but here I can see numerous unnecessary file changes.
> The snippets that you mentioned here, explain it.
> The absence of models.py as mentioned in your first approach is extremely telling. I would urge you to check out the documentation of the project and opensource in general before making redundant pull requests.
> Unnecessary changes and commits degrade the repository and will not be accepted. Keep this in mind while contributing to any repository in future.
> @this-is-yaash you can close this PR as it lacks the solution and is simply additional of redundant code.

Sorry but I think there's a misunderstanding...those commits aren't mine... I just ran git pull as mentioned by @this-is-yaash in my last pr...also the issue requirements got changed and I was unaware bout it...

File: README.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
 # SNEAKY PACKAGE 👾
-![GSSoC-ext](GSSoC-Ext.png)
 ## Table of Contents
-1. [🚀Project Overview](#project-overview)
-2. [✨ Key Features ✨](#✨key-features✨)
-3. [🚀 How to Contribute to This Project](#🚀-how-to-contribute-to-this-project)
```
[the remark this answers is not in the document]
Comment (ShashwatAgrawal20) [reply README.md:6]: Some README changes were required, but not all that he made.

[the remark this answers is not in the document]
Comment (techy4shri) [reply README.md:6]: Was it part of his assigned issue? Or was he promoted by either the PA or any of the mentors assigned to this project to edit Readme?

**Review by techy4shri — CHANGES_REQUESTED** [review]
Hello @Prakhar29Sharma
First and foremost, never edit files which are not part of your assigned issue. That is a violation ofcontribution guidelines in general and is considered point-farming.
As per your approach, you were to edit models.py file but here I can see numerous unnecessary file changes.
The snippets that you mentioned here, explain it.
The absence of models.py as mentioned in your first approach is extremely telling. I would urge you to check out the documentation of the project and opensource in general before making redundant pull requests.
Unnecessary changes and commits degrade the repository and will not be accepted. Keep this in mind while contributing to any repository in future.
@this-is-yaash you can close this PR as it lacks the solution and is simply additional of redundant code.

**Comment by techy4shri** [comment]
@Prakhar29Sharma if there is any change in requirement or anything, always mention it in your PR. That helps not only label assignment but also in reviewing.

**Comment by jindalpriyanshu101** [comment]
Heyi buddy @Prakhar29Sharma, it looks like latest changes and your changes resulted in a chaos after `git pull`, As we're not able to figure out your changes i suggest you to close this PR, and create a new one as all the file names are changed. 

For Your Information, i'm listing the changes below:
```
models.py -> show.py
graph.py -> clip.py
piechart.py -> write.py
clp.py (web scrapin) -> scrap.py
```

Therefore, kindly discard your changes, Sync the fork and utilise these file conventions in new PR.

**Pull request closed without merging**